packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/help-tip.twig

```diff
@@ -0,0 +1,47 @@
+<div
+  class="ma__help-tip mobile-tray ma__help-tip__text--c-primary"
+  id="{{ helpTip.id }}"
+>
+  <span class="ma__help-tip__label" id="label-{{ helpTip.id }}">
+    {% if helpTip.textBefore %}
+      <span>{{ helpTip.textBefore }}</span>
+    {% endif %}
+    <button
+      class="ma__help-tip__trigger {{ helpTip.expanded ? 'ma__help-tip__trigger--active' : '' }}"
+      id="trigger-{{ helpTip.id }}"
+      aria-expanded="{{ helpTip.expanded ? 'true' : 'false' }}"
+      aria-controls="help-tip-content-{{ helpTip.id }}"
+    >
+      <span>{{ helpTip.textTrigger }}</span>
+      <span class="ma_help-tip__label-a11y ma__visually-hidden">Open tooltip to learn more.</span>
+      {{icon('question-mark')}}
+    </button>
+    {% if textAfter %}
+      <span>{{ helpTip.textAfter }}</span>
+    {% endif %}
+  </span>
+  <div
+    class="ma__help-tip__container ma__help-tip__container--c-primary ma__help-tip__content {{ helpTip.expanded ? '' : 'collapsed' }}"
+    id="help-tip-content-{{ helpTip.id }}"
+    style="max-height:none"
+  >
+    <button
+      class="ma__help-tip__close-mobile"
+      aria-label="close tooltip"
```

Comment:
Based on above comment, this button doesn't make sense to screen reader users.  Let's remove L.30 aria-label.  It can be commented out for the future when interaction within aria-describedby content becomes available.